includes/Core/Authentication/Clients/Google_Proxy_Client.php

```diff
@@ -184,8 +219,11 @@ public function authorize( ClientInterface $http = null ) {
 					// Due to original callback signature this can only accept the token itself.
 					call_user_func( $callback, '', $creds['access_token'] );
 				}
-			} catch ( \Exception $e ) { // phpcs:ignore Generic.CodeAnalysis.EmptyStatement
-				// Ignore exceptions.
+			} catch ( Exception $e ) { // phpcs:ignore Generic.CodeAnalysis.EmptyStatement
```

Comment:
This is no longer an empty catch 😄 
```suggestion
			} catch ( Exception $e ) {
```